docs/modules/ROOT/pages/how-tos/testing/index.adoc

```diff
@@ -1 +1,33 @@
-= Testing your components and applications
\ No newline at end of file
+= Testing your components and applications
+
+You can ensure that your applications are stable, secure, compliant, and mutually compatible by implementing tests for {ProductName} to run on their components. There are currently 3 types of tests in {ProductName}:
+
+* Build-time tests
+* Custom tests
+* Integration tests
+
+The following sections explain each of these test types in greater detail.
+
+== Build-time tests
+
+Build-time tests in {ProductName} ensure the stability and security of the application, its components, its build pipeline, and the environment in which it is being tested. 
+
+The build-time tests used in {ProductName} are executed in the form of Tekton xref:glossary/index.adoc#task[tasks]. The utility used for validating container information is link:https://www.conftest.dev/[conftest]. A full listing of {ProductName} surface-level tests is available in this document: xref:./build/index.adoc[Build-time tests].
+
+Build-time tests include the following security-focused tests:
+
+* Vulnerability scanning via Clair
+* Anti-virus scanning via ClamAV
+* Code scanning via SAST tools
++
+** You can enable the Snyk test by following xref:./build/snyk.adoc[these instructions].
+
+== Custom tests
```

Comment:
These are custom build-time tests specifically. Not custom IntegrationTestScenarios. Do we want to add anything about the latter?

Reply:
Oh, hmm, I thought that the second type of custom tests mentioned in this section would be a custom ITS. Because you create a YAML file and it defines a test that runs on the whole snapshot. Is that not quite right?

I'll at least edit the second sentence, while we're still talking about custom build-time tests, to say, "To add a custom build-time test..." Hopefully that gives the doc enough clarity.

Reply:
Given what you said about the review in general, I'm gonna hit the green button. But if you'd like me to adjust anything, I'll be happy to reopen it and make any corrections we need.

Reply:
We can come back and split up that paragraph later. It isn't quite clear with a second reading. You are talking about two different types of custom tests in the same paragraph. The first is a built time test and the second is a custom ITS.

It might make sense to have two sections -- build-time and integration tests. In each section, we can provide directives for how to customize.